state.js

```diff
@@ -2,25 +2,99 @@
 const state = {
   isWorkInterval: true,
   timerIntervalId: 0,
-  todoList: [],
-  maxId: 0,
+  todoList: [
```

Comment:
```suggestion
  todoList: INITIAL_TODOS
```

nitpick: maybe cleaner to extract these initial TODOs if we're leaving them in ? (shortcut: select the array, right-click -> refactor -> extract to constant)

Reply:
I made it
![image](https://github.com/antondurinin/ToDo/assets/167963627/7d6b0606-106b-468f-81c8-d81cc7a0171e)